sqlserver/datadog_checks/sqlserver/queries.py

```diff
@@ -143,6 +143,37 @@
     ],
 }
 
+INDEX_USAGE_STATS_QUERY = {
+    "name": "sys.dm_db_index_usage_stats",
+    "query": """
+    SELECT
+         DB_NAME(ixus.database_id) as db,
+         CASE
+            WHEN ind.name IS NULL THEN 'HeapIndex_' + OBJECT_NAME(ind.object_id)
```

Comment:
Can you explain this?

Reply:
heap indexes don't have a name (it's some gibberish id) so we prepend this to be more readable.